README.md

```diff
@@ -126,17 +126,17 @@ You can add a global option to every Kinvey CLI command to get the described beh
 
    When used after a `kinvey-cli` command, shows its usage instructions.
 
-* `--host <host>`
+* `--instanceId <instance ID>`
 
-   Hostname of a Kinvey instance. It has a default value of `https://manage.kinvey.com/` which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated host name as either an absolute URI (`https://kvy-us2-manage.kinvey.com/`) or an instance name only (`kvy-us2`).
+   ID (e.g., `kvy-us2`) or full hostname (e.g., `https://kvy-us2-manage.kinvey.com/`) of a Kinvey instance. It has a default value of `kvy-us1` (or `https://manage.kinvey.com/`) which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated instance ID.
 
 * `--no-color`
 
     Disable colors.
     
 * `--output <json>`
```

Comment:
This should be "--output <format>" and in the description the allowed values should be listed and explained. The way it is now, it seems I have to write "--output <json>" or "--output <text>"